book.toml

```diff
@@ -12,15 +12,15 @@ edition = "2021"
 extra-watch-dirs = ["po"]
 
 [preprocessor.links]
```

Comment:
Can we not remote the whole `[preprocessor.links]` line now? Or do we implicitly disable the preprocessors because we enable our own custom ones?